File: 프로그래머스/0/181866. 문자열 잘라서 정렬하기/문자열 잘라서 정렬하기.py
def solution(myString):
    # 'x'를 기준으로 문자열을 분리합니다.
    split_str = myString.split('x')
    # 빈 문자열을 제거하고 사전순으로 정렬합니다.
    return sorted(filter(None, split_str))

File: 프로그래머스/0/181866. 문자열 잘라서 정렬하기/문자열 잘라서 정렬하기.java
import java.util.Arrays;

class Solution {
    public String[] solution(String myString) {
        String[] answer = {};
        return Arrays.stream(myString.split("x"))
                     .filter(s -> !s.isEmpty())   // 빈 문자열 제거
                     .sorted()                   // 사전순 정렬
                     .toArray(String[]::new);    // 배열로 변환
    }
}